examples/visual-question-answering/README.md

```diff
@@ -16,7 +16,7 @@ limitations under the License.
 
 # Visual Question Answering Examples
 
-This directory contains a script that showcases how to use the Transformers pipeline API to run visual question answering task on HPUs.
+The `run_pipeline.py` script showcases how to use the Transformers pipeline API to run visual question answering task on HPUs.
 
```

Comment:
@vidyasiv Please add a brief description of the openclip script too

Reply:
@skavulya I moved the description off the top level heading. It's a pretty subjective decision so feel free to push changes as you see fit.